source/_data/releases.ts

```diff
@@ -106,7 +106,7 @@ async function getLatestVersion(repo: string): Promise<string> {
 /**
  * Returns the version and URL for the latest release of all implementations.
  */
-export default async function (): Promise<Record<string, Release>> {
+module.exports = async function (): Promise<Record<string, Release>> {
```

Comment:
Can you add a comment here indicating why `export default` doesn't work? It's what we use everywhere else, so the difference here is conspicuous.

Reply:
Done, I wasn't sure if this is an issue with Eleventy or LiquidJS, but I documented it as an issue with Liquid

Reply:
I ended up saying it's Eleventy because AFAICT LiquidJS itself doesn't support JS/TS files as `_data`, that's an Eleventy's feature